src/plugins/reporter-bugzilla.c

```diff
@@ -76,6 +76,7 @@ int attach_file_item(struct abrt_xmlrpc *ax, const char *bug_id,
 struct bugzilla_struct {
     char *b_login;
     char *b_password;
```

Comment:
Is there any reason to keep these two around?

Reply:
I think we could drop it, but I was not 100% sure and I did not want to go full ballistic and remove a lot of code just yet. I will add a comment there that the field are no longer used and that we will drop them in future release.

Reply:
I think we can remove them right away as this API should not be public and it's not even versioned. I believe it's only used by `reporter-bugzilla` and nowhere else.

Reply:
Actually, looking at the code, `b_login` is still used when people call `reported-bugzilla` with -w option ("Add bugzilla user to CC list"). But `b_password` is indeed a dead weight. Removing it...